README.md

```diff
@@ -13,11 +13,11 @@ A customizable countdown component for React.
 You can either install the module via `npm` or `yarn`:
 
 ```
-npm install --save-dev react-countdown-now
+npm install react-countdown-now --save
 ```
 
 ```
-yarn add --dev react-countdown-now
+yarn add react-countdown-now --save
```

Comment:
`add ... --save` (https://yarnpkg.com/lang/en/docs/cli/add/#toc-yarn-add) doesn't exist in `yarn`, it's apparently adding the package to `dependencies` by default. Could you please remove this one?

Thank you for this!

Reply:
I removed, you're right! 😄